### PR TITLE
My Name Json File added

### PR DESCRIPTION
**Pull request by Ranamalsingh12**
This PR is for adding [Rana Mal Singh](https://github.com/Ranamalsingh12) Json. 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1584"></a>

### CODE REVIEW

**Comment by Ranamalsingh12** [comment]
This PR is to add [Rana Mal Singh](https://github.com/Ranamalsingh12)Json in public folder.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Profile data looks good to me! 

But please Revert the changes from `package-lock.json` file, then it's good to go! 🛠

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Comment by Ranamalsingh12** [comment]
Ok i will do it .

**Comment by Ranamalsingh12** [comment]
Hey @kumarsonsoff3 i have reverted the changes in the package-lock.json file. Can you check it now? and thanks for your response earlier

**Review by aparna2071 — APPROVED** [review]
Your code looks good to me

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM! Thanks for contributing! 🚀

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

File: public/data/Ranamalsingh12.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "url": "https://github.com/Ranamalsingh12",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/RanaMS999",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/Ranamalsingh12.json:18]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**